pallets/erc20/src/mock.rs

```diff
@@ -1,122 +1,116 @@
 #![cfg(test)]
 
-use super::*;
-
+use chainbridge as bridge;
 use frame_support::{ord_parameter_types, parameter_types, weights::Weight};
 use frame_system::{self as system};
-use sp_core::hashing::blake2_128;
-use sp_core::H256;
+pub use pallet_balances as balances;
+use sp_core::{hashing::blake2_128, H256};
 use sp_runtime::{
-    testing::Header,
-    traits::{AccountIdConversion, BlakeTwo256, Block as BlockT, IdentityLookup},
-    Perbill,
+	testing::Header,
+	traits::{AccountIdConversion, BlakeTwo256, Block as BlockT, IdentityLookup},
+	Perbill,
 };
 
+use super::*;
 use crate::{self as example, Config};
-use chainbridge as bridge;
-pub use pallet_balances as balances;
 
 parameter_types! {
-    pub const BlockHashCount: u64 = 250;
-    pub const MaximumBlockWeight: Weight = 1024;
-    pub const MaximumBlockLength: u32 = 2 * 1024;
-    pub const AvailableBlockRatio: Perbill = Perbill::one();
+	pub const BlockHashCount: u64 = 250;
+	pub const MaximumBlockWeight: Weight = 1024;
+	pub const MaximumBlockLength: u32 = 2 * 1024;
+	pub const AvailableBlockRatio: Perbill = Perbill::one();
 }
 
 impl frame_system::Config for Test {
-    type BaseCallFilter = ();
-    type Origin = Origin;
-    type Call = Call;
-    type Index = u64;
-    type BlockNumber = u64;
-    type Hash = H256;
-    type Hashing = BlakeTwo256;
-    type AccountId = u64;
-    type Lookup = IdentityLookup<Self::AccountId>;
-    type Header = Header;
-    type Event = Event;
-    type BlockHashCount = BlockHashCount;
-    type MaximumBlockWeight = MaximumBlockWeight;
-    type DbWeight = ();
-    type BlockExecutionWeight = ();
-    type ExtrinsicBaseWeight = ();
-    type MaximumExtrinsicWeight = ();
-    type MaximumBlockLength = MaximumBlockLength;
-    type AvailableBlockRatio = AvailableBlockRatio;
-    type Version = ();
-    type ModuleToIndex = ();
-    type AccountData = balances::AccountData<u64>;
-    type OnNewAccount = ();
-    type OnKilledAccount = ();
-    type SystemWeightInfo = ();
+	type BaseCallFilter = ();
+	type Origin = Origin;
+	type Call = Call;
+	type Nonce = u64;
+	type Block = u64;
+	type Hash = H256;
+	type Hashing = BlakeTwo256;
+	type AccountId = u64;
+	type Lookup = IdentityLookup<Self::AccountId>;
+	type Event = Event;
+	type BlockHashCount = BlockHashCount;
+	type MaximumBlockWeight = MaximumBlockWeight;
+	type DbWeight = ();
+	type BlockExecutionWeight = ();
+	type ExtrinsicBaseWeight = ();
+	type MaximumExtrinsicWeight = ();
+	type MaximumBlockLength = MaximumBlockLength;
+	type AvailableBlockRatio = AvailableBlockRatio;
+	type Version = ();
+	type ModuleToIndex = ();
+	type AccountData = balances::AccountData<u64>;
+	type OnNewAccount = ();
+	type OnKilledAccount = ();
+	type SystemWeightInfo = ();
 }
 
 parameter_types! {
-    pub const ExistentialDeposit: u64 = 1;
+	pub const ExistentialDeposit: u64 = 1;
 }
 
 ord_parameter_types! {
-    pub const One: u64 = 1;
+	pub const One: u64 = 1;
```

Comment:
for some reason formatting has changed from 1 tab to 2 tabs across multiple files